### ISSUE TITLE
Uncaught Error: Cannot find module '/usr/local/share/atom/resources/app.asar/node_modules/emissary'

### ISSUE THREAD

**Issue by bpousland** [Glavin001/atom-beautify#605]
[Enter steps to reproduce below:]

1. Under install packages saerch for atom-beautify
2. Click install to install atom-beautify

**Atom Version**: 1.1.0-beta1-63e8e81
**System**: Ubuntu 14.10
**Thrown From**: [atom-beautify](https://github.com/Glavin001/atom-beautify) package, v0.28.14

### Stack Trace

Uncaught Error: Cannot find module '/usr/local/share/atom/resources/app.asar/node_modules/emissary'

```
At module.js:336

Error: Cannot find module '/usr/local/share/atom/resources/app.asar/node_modules/emissary'
  at Module._resolveFilename (module.js:334:15)
  at Function.Module._resolveFilename (/usr/local/share/atom/resources/app.asar/src/module-cache.js:383:52)
  at Function.Module._load (module.js:284:25)
  at Module.require (module.js:363:17)
  at require (module.js:382:17)
  at Object.<anonymous> (/home/brad/.atom/packages/atom-beautify/src/beautify.coffee:447:16)
  at Object.<anonymous> (/home/brad/.atom/packages/atom-beautify/src/beautify.coffee:2:2)
  at Module._compile (module.js:428:26)
  at Object.defineProperty.value [as .coffee] (/usr/local/share/atom/resources/app.asar/src/compile-cache.js:169:21)
  at Module.load (module.js:353:32)
  at Function.Module._load (module.js:308:12)
  at Module.require (module.js:363:17)
  at require (module.js:382:17)
  at Package.module.exports.Package.requireMainModule (/usr/local/share/atom/resources/app.asar/src/package.js:725:34)
  at Package.module.exports.Package.activateConfig (/usr/local/share/atom/resources/app.asar/src/package.js:250:14)
  at PackageDetailView.module.exports.PackageDetailView.activateConfig (/usr/local/share/atom/resources/app.asar/node_modules/settings-view/lib/package-detail-view.js:212:26)
  at PackageDetailView.module.exports.PackageDetailView.completeInitialzation (/usr/local/share/atom/resources/app.asar/node_modules/settings-view/lib/package-detail-view.js:156:12)
  at PackageDetailView.module.exports.PackageDetailView.loadPackage (/usr/local/share/atom/resources/app.asar/node_modules/settings-view/lib/package-detail-view.js:168:21)
  at /usr/local/share/atom/resources/app.asar/node_modules/settings-view/lib/package-detail-view.js:298:17
  at Emitter.module.exports.Emitter.emit (/usr/local/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:86:11)
  at PackageManager.module.exports.PackageManager.emitPackageEvent (/usr/local/share/atom/resources/app.asar/node_modules/settings-view/lib/package-manager.js:628:27)
  at /usr/local/share/atom/resources/app.asar/node_modules/settings-view/lib/package-manager.js:417:26
  at exit (/usr/local/share/atom/resources/app.asar/node_modules/settings-view/lib/package-manager.js:56:16)
  at triggerExitCallback (/usr/local/share/atom/resources/app.asar/src/buffered-process.js:213:47)
  at ChildProcess.<anonymous> (/usr/local/share/atom/resources/app.asar/src/buffered-process.js:235:18)
  at emitTwo (events.js:87:13)
  at ChildProcess.emit (events.js:172:7)
  at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)

```

### Commands

```
  3x -7:27.3.0 core:move-up (atom-text-editor.editor.is-focused)
  3x -7:25.7.0 core:move-down (atom-text-editor.editor.is-focused)
     -7:17.8.0 editor:newline (atom-text-editor.editor.is-focused)
     -7:16.5.0 core:save (atom-text-editor.editor.is-focused)
  3x -7:15.9.0 core:move-left (atom-text-editor.editor.is-focused)
  4x -7:14.4.0 core:move-right (atom-text-editor.editor.is-focused)
 26x -7:12.1.0 core:move-up (atom-text-editor.editor.is-focused)
     -7:06.8.0 core:move-down (atom-text-editor.editor.is-focused)
  2x -7:06.4.0 core:move-right (atom-text-editor.editor.is-focused)
  2x -7:05.7.0 core:move-left (atom-text-editor.editor.is-focused)
     -7:04.5.0 core:move-right (atom-text-editor.editor.is-focused)
     -7:03.9.0 core:move-left (atom-text-editor.editor.is-focused)
     -7:03.4.0 core:move-right (atom-text-editor.editor.is-focused)
     -4:05.7.0 settings-view:open (atom-text-editor.editor.is-focused)
     -3:55.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
 22x -0:10.4.0 core:backspace (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "atom-dark-syntax"
    ],
    "disabledPackages": [
      "linter-js-standard"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-ternjs, v0.8.2
linter, v1.8.1
linter-gcc, v0.3.8

# Dev
No dev packages
```

**Comment by Glavin001** [comment]
Could you try uninstalling and reinstalling? That normally fixes this.

**Comment by m1ga** [comment]
should be this issue: https://github.com/Glavin001/atom-beautify/issues/428

**Comment by Glavin001** [comment]
Duplicate of #428. Thanks @m1ga

**Comment by Glavin001** [comment]
Published fix to v0.28.15

Thank you everyone for your patience.
